_tour/tuples.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: tour
+title: Tuples
+
+discourse: true
+
+partof: scala-tour
+
+num: 
+next-page: mixin-class-composition
+previous-page: traits
+topics: tuples
+
+redirect_from: "/tutorials/tour/tuples.html"
+---
+
+In Scala, a tuple is a class that can hold elements of different types.
+Tuples are immutable.
+
+Tuples come handy when we have to return multiple values from a function.
+
+A tuple can be created as:
+
+```tut
+val ingredient = ("Sugar" , 25)
+
+println(ingredient.getClass) // class scala.Tuple2 
+```
+This creates a tuple containing a String element and an Int element.
+
+Tuple in Scala is a series of classes: Tuple2, Tuple3, etc., through Tuple22.
+So when we create a tuple with n elements(n lying between 2 and 22), Scala basically instantiates
+one of the corresponding class from the group, parametrized with types of constituent elements.
+For eg., ingredient is of type Tuple2[String, int].
+
+## Accessing the elements
+
+Tuple elements are accessed using underscore syntax.
+tuple._n gives nth element(given there are that many elements).
+
+```tut
+println(ingredient._1) // Sugar
+
+println(ingredient._2) // 25
+```
+
+## Iterating a tuple
+
+Tuple provides productIterator method to iterate it.
+
+```tut
+ingredient.productIterator.foreach(println)
+```
+Tuple's toString method gives a String representation of its contents.
+
+```tut
+ingredient.toString // (Sugar,25)
+```
+
+Scala tuple also supports destructuring.
```

Comment:
bump this to a `##` header.

PS: I think it's kind of neat to mention that `()` is conceptually `Tuple0`, or `Unit`. There can only be one value of this type since it has no elements.

Maybe one final comment: you should use case classes whenever the elements have more meaning.